applications/ecallmgr/src/ecallmgr_originate.erl

```diff
@@ -768,6 +768,8 @@ maybe_send_originate_uuid({'fs', UUID}, Pid, #state{server_id=ServerId
                                                    }) ->
     CtlQ = gen_listener:queue_name(Pid),
     publish_originate_uuid(ServerId, UUID, JObj, CtlQ);
+maybe_send_originate_uuid({'api', UUID}, Pid, State) ->
+    maybe_send_originate_uuid({'fs', UUID}, Pid, State);
```

Comment:
@lazedo the output of dialyzer in CircleCI made me realize this can be written a bit differently. Functionality-wise it still stands that {'api', UUID} should result in the originate_uuid event being published